aten/src/ATen/native/TensorAdvancedIndexing.cpp

```diff
@@ -600,7 +609,17 @@ Tensor & masked_fill__cpu(Tensor& self, const Tensor & mask, const Tensor & valu
   TORCH_CHECK(value.dim() == 0, "masked_fill_ only supports a 0-dimensional value tensor, but got tensor "
       "with ", value.dim(), " dimension(s).");
 
-  masked_fill_impl_cpu(self, mask, value.item());
+  masked_fill_impl(self, mask, value.item());
+  namedinference::propagate_names_if_nonempty(self, maybe_outnames);
+  return self;
+}
+
+Tensor & masked_fill__cuda(Tensor& self, const Tensor & mask, const Tensor & value) {
```

Comment:
Please combine cpu and cuda versions, since they going to be automatically separated on kernel calls.

Reply:
Sure.